src/simplewallet/simplewallet.cpp

```diff
@@ -2386,6 +2388,43 @@ bool simple_wallet::stop_mining_for_rpc(const std::vector<std::string> &args)
   return true;
 }
 
+bool simple_wallet::show_qr_code(const std::vector<std::string> &args)
+{
+  uint32_t subaddress_index = 0;
+  if (args.size() >= 1)
+  {
+    if (!string_tools::get_xtype_from_string(subaddress_index, args[0]))
+    {
+      fail_msg_writer() << tr("invalid count: must be an unsigned integer");
```

Comment:
s/count/index/ ?

Reply:
thanks, fixed. looks like I copied the error message without reading them...